docs/book/how-to/advanced-topics/manage-zenml-server/best-practices-upgrading-zenml.md

```diff
@@ -67,7 +67,7 @@ loaded_artifact = artifact.load()
 While ZenML strives for backward compatibility, be prepared for occasional breaking changes (e.g., [the Pydantic 2 upgrade](https://github.com/zenml-io/zenml/releases/tag/0.60.0)).
 
 - **Changelog Review**: Always review the [changelog from new releases](https://github.com/zenml-io/zenml/releases) for new syntax, instructions, or breaking changes.
-- **Migration Scripts**: Use provided [migration scripts](../../how-to/manage-the-zenml-server/migration-guide/migration-guide.md) when available to handle database schema changes.
+- **Migration Scripts**: Use provided [migration scripts](migration-guide/migration-guide.md) when available to handle database schema changes.
```

Comment:
is this correct?

Reply:
yes, the migration guide is in the same directory and resolves to the right file